server.js

```diff
@@ -107,10 +104,9 @@ const {
 const {
   NavYmlService,
 } = require("@services/fileServices/YmlFileServices/NavYmlService")
-const { initializeIdentityServices } = require("@services/identity")
+const { usersService } = require("@services/identity")
```

Comment:
Nit: Use PascalCase to keep consistent with the rest of the services/routers

Reply:
We use PascalCase for the Classes (e.g. MediaCategoriesRouter) and camelCase for the instances (e.g. mediaDirectoryV2Router). This is the same across routers and services

Reply:
actually then it should be snakeCased as this (`userService`) is an instance ._.